.github/workflows/publish-charts.yaml

```diff
@@ -0,0 +1,88 @@
+name: publish-charts
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository from where we want to build Charts'
+        required: true
+      repo_ref:
+        description: 'The branch/tag that will be used as new version for charts. Usually tag ref'
+        required: true
+      repo_chart:
+        description: 'Object with source path, destination path and chart name'
+        required: true
+      currentChartVersion:
+        description: 'Chart version set in `Chart.yaml`. Default: `LATEST`'
+        required: false
+        default: 'LATEST'
+
+env:
+  REPO: ${{ github.event.inputs.repo }}
+  REPO_REF: ${{ github.event.inputs.repo_ref }}
+  CurrentChartVersion: ${{ github.event.inputs.currentChartVersion }}
```

Comment:
I would rename this variable to CURRENT_CHART_VERSION to keep things consistent.  Unless there is a specific need to keep the name like this

Reply:
yep! makes sense, somehow it was left like this during the testing phase thanks!